app/models/disk.rb

```diff
@@ -75,4 +77,8 @@ def base_storage_extents
   def storage_systems
     miq_cim_instance.nil? ? [] : miq_cim_instance.storage_systems
   end
+
+  def used_disk_storage
```

Comment:
@NickLaMuro whether you end up using `virtual_aggregate` or not, would prefer keeping the column logic locally in a `virtual_column` / `virtual_attribute. (huh, I defined this as a `virtual_column`. Odd)